pom.xml

```diff
@@ -151,8 +151,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.2</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
```

Comment:
This is a very significant change，Many JDK8 projects may not be able to upgrade dependencies.
I think this will reduce compatibility, it seems unnecessary.